src/modules/apiHelper.js

```diff
@@ -5,8 +5,8 @@ const Article = {
   async create(article) {
     article["article"]["author_ids"] = [];
     const params = article;
-    const headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"))
-    const response = await axios.post("/api/articles/", params, headers);
+    const headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+    const response = await axios.post("/api/articles/", { params, headers });
```

Comment:
Is there a reason we have to wrap the params and header arguments as an object?

Reply:
The authorization wasn't working on Netlify after the last merge, so I added {} to be inline with the end user which was authorizing